reframe/core/schedulers/slurm.py

```diff
@@ -68,6 +69,17 @@ class SlurmJob(sched.Job):
     # standard job state polling using sacct.
     SACCT_SQUEUE_RATIO = 10
 
+    @staticmethod
+    def _get_nodes_from_description(descriptions):
+        nodes = set()
+        if descriptions:
```

Comment:
Why do you need this? Can `descriptions` be `None`? Cos only then this has a meaning.